### PR TITLE
Home (Partially Done)

### PR DESCRIPTION
**Pull request by jevenson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bootstrap": "^4.0.0"
+    "bootstrap": "^4.0.0",
+    "jquery": "^3.3.1",
```
Comment (kylevogt) [inline package.json]: REEEEEEEEEEEEEEE

File: src/site/admin/config.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 backend:
   name: git-gateway
-  branch: master
+  branch: home
```
Comment (jevenson) [inline src/site/admin/config.yml]: rename to master

File: src/site/styles/_home-header.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    .conference-date {
+      margin-right: 20px;
+    }
+
+    .conference-times {
+      margin-left: 20px;
+    }
+  }
+
+  svg {
```
Comment (jevenson) [inline src/site/styles/_home-header.scss]: give better class name for svg

File: package.json (hunk 1 above)
Comment (jevenson) [reply package.json]: i know :(

**Pull request merged**